ThreeBotPackages/tfgrid/workloads/actors/workload_manager.py

```diff
@@ -372,7 +388,15 @@ def workloads_list(self, node_id, cursor, schema_out, user_session):
         ```
         """
         output = schema_out.new()
-        reservations = self._filter_reservations(node_id, ["deploy", "delete"], cursor)
+        reservations_subset = self._filter_reservations(node_id, ["deploy", "delete"], cursor)
```

Comment:
I think this should be on `["deploy"]` since all ones in the 'delete' state will be in the actionable reservations queue anyway